docs/usage/control-plane-endpoints-and-ports.md

```diff
@@ -0,0 +1,46 @@
+# Endpoints and Ports of a Shoot Control-Plane
+
+With the reversed VPN tunnel, there are no endpoints with open ports in the shoot cluster required by gardener.
+In order to allow communication to the shoots control-plane in the seed cluster, there are shared endpoints for all shoots of a seed cluster.
+The main entry-point into the seed cluster is the loadbalancer of the Istio ingress-gateway service. Depending on the infrastructure provider, there can be one IP address per zone.
+
+The Istio ingress-gateway exposes the following TCP ports:
+
+* **443** for requests to the shoot API-server. The request is dispatched according to the set TLS SNI extension.
+* **8443** for requests to the shoot API-server via `api-server-proxy`. 
+* **8132** to establish the reversed VPN connection.
+
+## Kube API-server via SNI 
+
+![API-server via SNI](./images/api-server-sni.png)
+
+DNS entries for `api.<external-domain>` and `api.<shoot>.<project>.<internal-domain>` point to the loadbalancer of an Istio ingress-gateway service.
+The kubernetes client sets the server name to `api.<external-domain>` or `api.<shoot>.<project>.<internal-domain>`.
+Based on SNI, the connection is forwarded to the respective API-server at TCP layer. There is no TLS termination at the Istio ingress-gateway.
+TLS termination happens on the shoots API-server. Traffic is end-to-end encrypted between the client and the API-server. The certificate authority and authentication are defined in the corresponding `kubeconfig`.
+Details can be found in [GEP-08](../proposals/08-shoot-apiserver-via-sni.md).
+
+## Kube API-server via api-server-proxy
+
+![API-server-proxy](./images/api-server-proxy.png)
```

Comment:
Please adapt the picture:
`kubernetes.default.cluster.local` => `kubernetes.default.svc.cluster.local`
`kube-api-server.<shoot>.svc` => `kube-apiserver.<shoot>.svc`